tests/performance/test_api/test_api.py

```diff
@@ -23,7 +38,9 @@ def test_api_endpoints(test_case, test_configuration, set_api_test_environment,
     """Make an API request for each `test_case`. `test_configuration` fixture is only used to add metadata to the
```

Comment:
Configuration ids are not properlly set, resulting in a not very verbose output:
```
FAILED tests/performance/test_api/test_api.py::test_api_endpoints[test_case25-test_configuration0] - assert 400 == 200
FAILED tests/performance/test_api/test_api.py::test_api_endpoints[test_case32-test_configuration0] - assert 1 == 0
```

Maybe after removing configuration parametrization, the following will do the job:
```
case_ids = [f"{case['method']}_{case['endpoint']}" for case in test_data['test_cases']]

# Tests
@pytest.mark.parametrize('test_case', test_data['test_cases'], ids=case_ids)

```

Producing the following output:


```
tests/performance/test_api/test_api.py::test_api_endpoints[get_/cluster/local/info-] PASSED                                                                                                                 [  1%]
tests/performance/test_api/test_api.py::test_api_endpoints[get_/cluster/nodes-] PASSED                                                                                                                      [  3%]
tests/performance/test_api/test_api.py::test_api_endpoints[get_/cluster/healthcheck-] PASSED                                                                                                                [  5%]
tests/performance/test_api/test_api.py::test_api_endpoints[get_/cluster/status-] PASSED                                                                                                                     [  7%]
tests/performance/test_api/test_api.py::test_api_endpoints[get_/cluster/local/config-] PASSED   
```